vita3k/kernel/src/sync_primitives.cpp

```diff
@@ -98,6 +98,14 @@ inline int handle_timeout(const ThreadStatePtr &thread, std::unique_lock<std::mu
             queue->erase(data_it);
 
             return RET_ERROR(SCE_KERNEL_ERROR_WAIT_TIMEOUT);
+        } else {
+            auto end = std::chrono::steady_clock::now();
+            uint32_t real_timeout = (end - start) / std::chrono::microseconds(1);
```

Comment:
```suggestion
            uint32_t real_timeout = static_cast<uint32_t>(std::chrono::duration_cast<std::chrono::milliseconds>(end - start).count());
```

Reply:
Thank you. I changed duration in your suggestion to microseconds